staging/src/k8s.io/kubectl/pkg/cmd/delete/delete_flags.go

```diff
@@ -34,7 +34,6 @@ type DeleteFlags struct {
 
 	All            *bool
 	AllNamespaces  *bool
-	Cascade        *bool
```

Comment:
I think we want a metav1.DeletePropagation here?

Reply:
I moved `metav1,DeletePropagation` to DeleteOptions. Here we use `string` to get value from Cobra command :)